<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar una salida</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dinero</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Cantidad de dinero a extraer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsbMonto">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="prefix">
            <string>$ </string>
           </property>
           <property name="maximum">
            <double>99999999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Motivo o raz√≥n:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="teMotivo"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Producto/Autoconsumo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Seleccione un producto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leProducto">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btBuscarProducto">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Cantidad actual</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbCantidad">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Cantidad a extraer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbCantidad">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>72</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>ESC Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../GUI/images_rc/icons.qrc">
         <normaloff>:/newPrefix/stock_close.png</normaloff>:/newPrefix/stock_close.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btRegistroSalidas">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Registro de salidas</string>
       </property>
       <property name="icon">
        <iconset resource="../GUI/images_rc/icons.qrc">
         <normaloff>:/newPrefix/xfce-filemanager.png</normaloff>:/newPrefix/xfce-filemanager.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btGuardar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>F9 Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../GUI/images_rc/icons.qrc">
         <normaloff>:/newPrefix/computer.png</normaloff>:/newPrefix/computer.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>dsbMonto</tabstop>
  <tabstop>teMotivo</tabstop>
  <tabstop>btGuardar</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btRegistroSalidas</tabstop>
  <tabstop>leProducto</tabstop>
  <tabstop>btBuscarProducto</tabstop>
  <tabstop>sbCantidad</tabstop>
 </tabstops>
 <resources>
  <include location="../GUI/images_rc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btCancelar</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
